.github/workflows/build_deployment.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         node-version: 12.x
     - name: Clean install service dependencies
       run: npm ci
+    - name: Unit tests
+      run: npm run test
```

Comment:
In this case, it would work but not if we have custom scripts.. check this [answer](https://stackoverflow.com/a/51358329).. For consistency with our future custom scripts, we could keep the keyword `run` in commands, lmk what you think?